File: 1001-1010/1005 Spell It Right (20).py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

Endlish_number= {0:"zero",  1:"one",
                 2:"two",   3:"three",
                 4:"four",  5:"five",
                 6:"six",   7:"seven",
                 8:"eight", 9:"nine",}

def main():
    number = int(input(""))
    sum = 0
    while(number != 0):
        sum = sum + number % 10
        number = number // 10

    sum_str = str(sum)

    for i in range(len(sum_str)):
        print(Endlish_number[int(sum_str[i])],end="")
        if(i != (len(sum_str) - 1)):
            print(" ",end="")


if __name__ == "__main__":
    main()

File: Practice/1001-1010/1005 Spell It Right (20).cpp
#include "stdio.h"

int sum_dight(int number);

int main(void)
{
	int num,sum;
	char sum_in_English[10] = ["zero","one","two","three","four","five","six","seven","eight","nine"];
	//scanf("%d",&num);
	num = 12345;
	sum = sum_dight(num);
	while(sum != 0)
	{
		printf("%c",sum_in_English[sum % 10]);
	}
	printf("%d",sum);
	return 0;
} 


int sum_dight(int number)
{
	int sum = 0;
	while(number != 0)
	{
		sum = sum + number % 10;
		number = number / 10;
	}
	return sum;
}
